### DISCUSSION TITLE
科研过滤器的文档

### DISCUSSION THREAD

**Discussion by ChthoIly** [LmeSzinc/AzurLaneAutoScript#2087] [Q&A] [answered]
没找到科研过滤器的文档在哪，我记得是有一个文档解释科研过滤器的关键词的。有没有人能给我贴个链接谢谢

**Comment by bookbug666** [comment, marked as answer]
https://github.com/LmeSzinc/AzurLaneAutoScript/wiki/filter_string_cn
